crates/editor/src/element.rs

```diff
@@ -1366,6 +1366,44 @@ impl EditorElement {
                 }
             }
 
+            if layout.is_singleton && scrollbar_settings.symbols_selections {
```

Comment:
Above here, there's a check for 'show_scrollbars', that wasn't updated to incorporate this new information. Could you add a branch to resolution of the `ShowScrollbar::Auto` setting, on line 2066?